lib/reporters/mini.js

```diff
@@ -11,9 +11,9 @@ var repeating = require('repeating');
 var colors = require('../colors');
 
 chalk.enabled = true;
-Object.keys(colors).forEach(function (key) {
+for (let key of Object.keys(colors)) {
 	colors[key].enabled = true;
-});
+}
```

Comment:
You mean adding a MiniReporter.prototype.enabled = function () {} that would enable all the colors?